docs/docs/Configuration/api-keys-and-authentication.mdx

```diff
@@ -138,33 +147,37 @@ You can use the [`.env.example`](https://github.com/langflow-ai/langflow/blob/ma
 
 ### LANGFLOW_AUTO_LOGIN {#langflow-auto-login}
```

Comment:
@jordanrfrazier could you please review line 148-180 again, and please confirm that it is correct?

IIRC, LANGFLOW_AUTO_LOGIN doesn't control authentication for the API - only the visual editor and CLI - correct?